src/hotspot/share/gc/z/zStat.cpp

```diff
@@ -760,8 +760,10 @@ ZStatCriticalPhase::ZStatCriticalPhase(const char* name, bool verbose) :
     _verbose(verbose) {}
 
 void ZStatCriticalPhase::register_start(const Ticks& start) const {
```

Comment:
So this method is empty.
Shall we just remove it?
Thanks.

Reply:
No, because it's pure virtual in the super class.